src/test/utils/options/options.test.ts

```diff
@@ -25,6 +26,43 @@ describe('Options', () => {
         options.removeDeclarationByName(declaration.name);
     });
 
+    it('Does not throw if number declaration has no min and max values', () => {
+        const declaration: NumberDeclarationOption = {
+            name: 'test-number-declaration',
+            help: '',
+            type: ParameterType.Number,
+            defaultValue: 1
+        };
+        doesNotThrow(() => options.addDeclaration(declaration));
```

Comment:
doesNotThrow is redundant. All it does is rethrow an error if one occurs.

Reply:
I'm confused. What would be the right solution? I guess you understand what it is trying to assert.

Reply:
It doesn't hurt anything, I guess there is some semantic value, but that's provided by the test name. It's fine as is.